recipes/sample-sheet/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "sample-sheet" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 83f486f592db1275d8435341638d56dd2ed6b0ad2a911dfd7786ece6310de309
+
+build:
+  number: 0
+  skip: True # [not py36]
+
+requirements:
+  host:
+    - click
+    - pip
+    - python
+    - smart_open >=1.5.4
+    - tabulate
+    - terminaltables
+  run:
+    - click
+    - python
```

Comment:
... here.

According to your line above, you'd use `3.6.*`. Can you check if it works with `3.7.*` as well? If it does, writing `>=3.6` would be the correct constraint.